runtime/doc/lua.txt

```diff
@@ -876,6 +876,32 @@ vim.region({bufnr}, {pos1}, {pos2}, {type}, {inclusive})       *vim.region()*
         whether the selection is inclusive or not, into a zero-indexed table 
         of linewise selections of the form `{linenr = {startcol, endcol}}` .
 
+                                             *vim.register_keystroke_callback()*
+vim.register_keystroke_callback({ns_id}, {fn})
+        Register a lua {fn} with an {ns_id} to be run after every keystroke.
+
+        NOTE: {fn} will be automatically removed if an error occurs while
+        calling.
```

Comment:
It would be tremendously getting messages for every keystroke if you made a bug, and people could wrap it in pcall, I suppose. I'm ok with this, but I would consider adding a sentence about the motivation (but not required).